src/gc/impl/conservative/gc.d

```diff
@@ -3090,10 +3292,10 @@ struct Pool
         {
             freebits.Dtor();
         }
-        finals.Dtor();
-        structFinals.Dtor();
-        noscan.Dtor();
-        appendable.Dtor();
+        finals.Dtor(false);
```

Comment:
if allocations don't explicitly set the default parameter, Dtor shouldn't do either.